src/TestUtils/GcloudWrapper/GcloudWrapperTrait.php

```diff
@@ -68,14 +65,15 @@ private function errorLog($message)
 
     protected function execWithRetry($cmd, $retries = 3, &$output = null)
     {
-        for ($i = 0; $i <= $retries; $i++) {
+        for ($i = 0; $i <= $retries; ++$i) {
```

Comment:
not that I mind, but why make this change? It won't make a difference

Reply:
IDE rewrite from before I configured php-cs-fixer to this repo configuration.